doc/reference/modules/transactions.xml

```diff
@@ -604,22 +604,22 @@ finally
                     The session will enlist with the first scope in which the session is used (or opened).
                     As of NHibernate v5.0, it will enlist its connection in the transaction regardless of
                     connection string <literal>Enlist</literal> setting. Prior to v5.0, it was relying on
-                    that setting being considered <literal>true</literal>. If the connection was supplied
-                    to the session at its opening, the session will not handle the connection enlistment:
-                    when supplying a connection to the session, it is your responsibility to handle its
-                    transaction enlistment.
+                    that setting being considered <literal>true</literal>, and on acquiring the connection
```

Comment:
Was not true. In fact this PR enlist the supplied connection too, not only owned connection.